### PR TITLE
awscli 2.8.2

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

File: Formula/awscli.rb [hunk 1]
```diff
@@ -29,8 +29,8 @@ class Awscli < Formula
   # 2. Ignore `six`. Update all other PyPI packages
 
   resource "awscrt" do
-    url "https://files.pythonhosted.org/packages/69/5f/997b6c9a8f77cf5f2f1e434a35088a4ddd119254b1a6dc8fb196e0607d66/awscrt-0.14.3.tar.gz"
-    sha256 "9cb96574ee28b7258a9fdd4beaf0a2061f5cb29945acdf2d88ad6d7de8a4e98e"
+    url "https://files.pythonhosted.org/packages/51/1f/4d9193f7b16d102c3c11d8edcb11ea741ac6562d8fcf4d1ae1aa223f197c/awscrt-0.14.0.tar.gz"
```
Comment (branchvincent) [inline Formula/awscli.rb:32]: https://github.com/aws/aws-cli/blob/2.7.32/setup.cfg#L39

Comment (chenrui333) [reply Formula/awscli.rb:32]: maybe we can just stick with the latest patch version
```
    Running setup.py install for awscrt: finished with status 'error'
  error: legacy-install-failure
```

Comment (chenrui333) [reply Formula/awscli.rb]: interesting, failed to build 0.14.3 as well

Comment (branchvincent) [reply Formula/awscli.rb]: weird that `ld` doesn't seem to print any actual warning, just:

```
ld: fatal warning(s) induced error (-fatal_warnings)
```

debugging locally, I can get it to build by dropping `-fatal_warnings`. not sure what else to try

Comment (chenrui333) [reply Formula/awscli.rb:32]: Weird, I actually did not have any issues of building it in my local.

**Comment by chenrui333** [comment]
looks like regression build with Rust 1.64.0 also failed.

**Comment by chenrui333** [comment]
At this point, should we create an upstream ticket to track down the build issue?

**Comment by cho-m** [comment]
If only Monterey is failing, I wonder if it is an XCode 14 linker bug given we hit linker issues when building other formulae (e.g. GCC). If anyone has local XCode beta release, could try and see if it helps fix build.

In that case, we would need to wait for next XCode release. I don't think we use alternative linkers unlike alternative compilers. Though it may build if you force `llvm`'s linker via `LDFLAGS=-fuse-ld=lld`.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> If only Monterey is failing, I wonder if it is an XCode 14 linker bug given we hit linker issues when building other formulae (e.g. GCC). If anyone has local XCode beta release, could try and see if it helps fix build.

do we have some issue to track that?

**Pull request closed without merging**